src/app/app.module.ts

```diff
@@ -9,12 +9,14 @@ import { AppRoutingModule } from "./app-routing.module";
 import { AppComponent } from "./app.component";
```

Comment:
```suggestion
import { AppComponent } from "./app.component";

```

Reply:
that's an internal import, so i think it's ok without the extra space